src/app/datagrid/selection/selection.html

```diff
@@ -52,11 +72,9 @@
     <clr-dg-column>Favorite color</clr-dg-column>
 
     <clr-dg-row *clrDgItems="let user of users" [clrDgItem]="user">
-        <clr-dg-action-overflow>
-            <ul class="list-unstyled">
-                <li><button type="submit" class="btn btn-sm btn-link">Edit</button></li>
-                <li><button type="submit" class="btn btn-sm btn-link">Delete</button></li>
-            </ul>
+        <clr-dg-action-overflow [clrDgActionItem]="user">
+            <a class="action-item" href="#" (click)="onEdit()" clrDgActionOverflowToggle>Edit</a>
+            <a class="action-item" href="#" (click)="onDelete()" clrDgActionOverflowToggle>Delete</a>
```

Comment:
Thanks for the good suggestion. Since the action menu is a part of the actionMenuHost, I achieved this by looking for the class name of the overflow menu container in the target (and its parents), and hiding it if found. Let me know if this is a clean enough a solution.